Messenger/Messenger.Core/Services/MessageService.cs

```diff
@@ -192,13 +192,23 @@ public static async Task<bool> DeleteMessage(uint messageId)
             LogContext.PushProperty("SourceContext", "MessageService");
             logger.Information($"Function called with parameters messageId={messageId}");
 
+            // Delete replies to prevent errors
             var replies = await RetrieveReplies(messageId);
 
             foreach (var reply in replies)
             {
                 await DeleteMessage(reply.Id);
             }
 
+            // Delete reactions to prevent errors
+            var reactions = await RetrieveReactions(messageId);
+
```

Comment:
could you add null checks here? foreach would throw error if `reactions` == null

Reply:
Fixed in 07a20b9